core/gui/src/app/workspace/service/operator-menu/operator-menu.service.ts

```diff
@@ -69,7 +66,9 @@ export class OperatorMenuService {
       this.workflowActionService.getJointGraphWrapper().getJointGroupHighlightStream(),
       this.workflowActionService.getJointGraphWrapper().getJointGroupUnhighlightStream()
     ).subscribe(() => {
-      this.effectivelyHighlightedOperators.next(this.getEffectivelyHighlightedOperators());
+      this.effectivelyHighlightedOperators.next(
```

Comment:
do we still need "effectivelyHighlighted" term?

Reply:
Renamed to `highlighted`.